src/generator/documentation.jl

```diff
@@ -14,15 +9,19 @@ function print_documentation(io::IO, cursor::CLCursor, indent, options, members:
     ids = get(options, "DAG_ids", Dict())
 
     if extract_c_comment_style == "doxygen"
-        doc = format_doxygen(cursor, options, members)
+        doc = format_doxygen(node.cursor, options, members)
     elseif extract_c_comment_style == "raw"
-        doc = format_raw(cursor, members)
+        doc = format_raw(node.cursor, members)
     else
-        return
+        if (callback_documentation = get(options, "callback_documentation", nothing)) ≢ nothing
+            doc = callback_documentation(node)
```

Comment:
This config entry can not be put in the toml config file, right?

Reply:
No, it has to be a callback.